### DISCUSSION TITLE
关于账号系统与隐藏皮肤

### DISCUSSION THREAD

**Discussion by Anruicat** [Meloong-Git/PCL#1106] [提问] [answered]
可不可以添加一个简单的账号系统，可以像QQ一样在另一台设备上登录老的设备就会退出登录，应该可以简单的防止一个账号7、8个人使用。另外可以将隐藏皮肤与账号关联，登录账号就可以使用已解锁的皮肤。
如果需要单独建立一个服务器，那么就开支太大、不值得了。

**Comment by Joker2184** [comment, marked as answer]
需要单独建立服务器
